src/common/obj_bencher.cc

```diff
@@ -799,6 +799,11 @@ int ObjBencher::seq_read_bench(int seconds_to_run, int num_objects, int concurre
   }
 
   runtime = ceph_clock_now() - data.start_time;
+  if(runtime <= 1) {
```

Comment:
> if the running time of this command is less than 1s

@PCzhangPC I was wondering whether this `1s` will change to `2s` in another execution in another machine with a different performance.  Have you tried anything like that? Is this `1s` depends on your execution environment?